### PR TITLE
Fixes wallet dropdown bug

### LINKED ISSUE #689: dropdown selected account bug
When a wallet is selected from the dropdown menu, and a transaction is selected to view the tx details, returning to the main transaction page, the wallet selection is reverted back to the default.

What is expected that is that the wallet dropdown should retain the initially selected wallet before the transaction was selected to view the details.

ezgif com-gif-maker(13)

### PR DESCRIPTION
**Pull request by AdimekweEbuka**
Fixes #689
godcr (testnet) 2021-11-20 13-44-17

### CODE REVIEW

**Review by devchoplife — CHANGES_REQUESTED** [review]
This fix is not implemented on tickets list page

**Review by devchoplife — DISMISSED** [review]
Looks good

File: ui/load/wallet.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ type WalletLoad struct {
 
 	SelectedWallet  *int
 	SelectedAccount *int
-}
+} //
```
Comment (Sirmorrison) [inline ui/load/wallet.go]: Why is the `//`?

File: ui/page/transaction/transaction_details_page.go [hunk 2]
```diff
@@ -104,7 +104,7 @@ func NewTransactionDetailsPage(l *load.Load, transaction *dcrlibwallet.Transacti
 			}
 		}
 	}
-
+	//
```
Comment (Sirmorrison) [inline ui/page/transaction/transaction_details_page.go]: What is this for?

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

File: ui/page/transaction/transaction_details_page.go (hunk 2 above)
Comment (AdimekweEbuka) [reply ui/page/transaction/transaction_details_page.go]: Sorry, forgot to take it out.

**Review by Sirmorrison — APPROVED** [review]

**Pull request merged**